src/Makefile

```diff
@@ -278,6 +274,14 @@ ifneq ($(OS), WINNT)
 	@ln -sf libjulia-debug.$(JL_MAJOR_MINOR_SHLIB_EXT) $(build_shlibdir)/libjulia-debug.$(JL_MAJOR_SHLIB_EXT)
 	@ln -sf libjulia-debug.$(JL_MAJOR_MINOR_SHLIB_EXT) $(build_shlibdir)/libjulia-debug.$(SHLIB_EXT)
 endif
+ifeq ($(USE_POLLY_ACC), 1)
+# Place libGPURuntime to be accessible under libjulia's RPATH
+ifeq ($(OS), WINNT)
+	@cp $(shell $(LLVM_CONFIG_HOST) --libdir)/libGPURuntime.dll $(build_shlibdir)
+else
+	@cp $(shell $(LLVM_CONFIG_HOST) --libdir)/libGPURuntime.$(SHLIB_EXT) $(build_shlibdir)
```

Comment:
`SHLIB_EXT` is set somewhere around line 500 of `Make.inc`. `Make.inc` is the file that defines the standard stuff that's used in all of the various makefiles, `SHLIB_EXT` being one example.

In this case, the variable is set using the eager resolution assignment operator, `:=`. Regular `=` does lazy resolution in Make. It also has a space before the assignment. So grepping with a trailing `=` won't find it. Just now I found it with `sift -e '\bSHLIB_EXT\s*[:?]?=' -n Make.inc`.